third_party/github.com/bazelbuild/rules_typescript/internal/tsc_wrapped/worker.ts

```diff
@@ -14,8 +14,9 @@ if (typeof global.gc !== 'function') {
 /**
  * Whether to print debug messages (to console.error) from the debug function
  * below.
+ * To see debug logs, run build with --define=VERBOSE_LOGS=1
  */
-export const DEBUG = false;
+export const DEBUG = process.env.VERBOSE_LOGS === '1';
```

Comment:
in other places, we just check that this variable is defined, not what value it has.
we do document that you ought to set the value to 1, but it's better to be permissive in what we accept, and be consistent

Reply:
Make sense, thanks!